packages/database/src/eventsDatabase/eventsDatabase.ts

```diff
@@ -36,47 +36,17 @@ try {
     CaptchaEvent = mongoose.model('CaptchaEvent', captchaEventSchema)
 }
 
-const addCaptchaEventRecord = async (record: StoredEventRecord): Promise<void> => {
-    try {
-        const newRecord = new CaptchaEvent(record)
-        await newRecord.save()
-        logger.info('Record added successfully')
-    } catch (error) {
-        logger.error('Error adding record to the database:', error)
-    }
-}
-
 export const saveCaptchaEvent = async (events: StoredEvents, accountId: string, atlasUri: string) => {
-    return new Promise((resolve, reject) => {
-        const connection = mongoose.createConnection(atlasUri)
-        for (let attempt = 1; attempt <= MAX_RETRIES; attempt++) {
-            connection.once('open', resolve).on('error', (e) => {
-                logger.warn(`Mongoose connection error`)
-                logger.error(e)
+    await mongoose
+        .connect(atlasUri)
+        .then(() => console.log('Connected to MongoDB Atlas'))
+        .catch((err) => console.error('Error connecting to MongoDB:', err))
```

Comment:
this will catch the error and proceed onwards. Is this what you intended?

Reply:
Yeah I don't want the provider to fall over just because it can't emit user events to mongo

Reply:
won't the next lines fallover because the db isn't connected?